ci/localstack/1-setup-queue.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo
+
+export AWS_ACCESS_KEY_ID=UNSET
+export AWS_SECRET_ACCESS_KEY=UNSET
+export AWS_DEFAULT_REGION=eu-west-2
+
+## make sure that localstack is running in the pipeline
+until curl http://localstack:4566/health --silent | grep -q "\"sqs\": \"available\""; do
+   sleep 5
+   echo "Waiting for LocalStack to be ready..."
+done
+
+awslocal sqs create-queue --queue-name search-queue-dlq
```

Comment:
This only creates the search queue - it looks like it needs the audit queues to exist to?